src/macros.rst

```diff
@@ -1044,37 +1044,33 @@ environment.
 Hygiene is categorized as follows:
```

Comment:
"Hygiene" needs to be defined in the Glossary, and all mentions to the term converted to references.